mailbox/jpa/src/test/java/org/apache/james/mailbox/jpa/mail/JPAMapperProvider.java

```diff
@@ -124,6 +134,6 @@ private EntityManagerFactory createEntityManagerFactory() {
 
     @Override
     public List<Capabilities> getNotImplemented() {
```

Comment:
That's funny, usually we do the opposite for a good reason : when we add a capability to the system, all classes that don't implement it yet don't suddenly break.